### PR TITLE
testing main branch protection rule

### PR DESCRIPTION
**Pull request by CeliaVerdugo**

### CODE REVIEW

**Comment by CeliaVerdugo** [comment]
just wanted to check if code owner was requested for a review before being able to merge which it did , meaning that the main branch protection rule worked

**Comment by fsantanar** [comment]
> just wanted to check if code owner was requested for a review before being able to merge which it did , meaning that the main branch protection rule worked

Great News! I will then review, approve and merge

**Review by fsantanar — APPROVED** [review]

**Pull request merged**
